packages/multichain-network-controller/CHANGELOG.md

```diff
@@ -7,4 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-[Unreleased]: https://github.com/MetaMask/core/
+## [1.0.0]
```

Comment:
We should not make it a major one for now.

We know we will have some fixes already, I would just go with `0.1.0` for now

Reply:
Updated